### PR TITLE
[WEB-1957] - Academy Landing Page 

### PR DESCRIPTION
**Pull request by RavenHursT**
https://goodparty.atlassian.net/browse/WEB-1957

### CODE REVIEW

File: app/(landing)/local-elections-webinar/components/LocalElectionsHero.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              alt="FWD Party Logo"
+            />
+            <span className="uppercase font-semibold text-sm align-middle lg:text-2xl">
+              Forward Party
+            </span>
+          </h6>
+        </div>
+        <div className="col-span-7 hidden lg:grid">
+          <Image
+            className="my-auto"
```
Comment (tomer-tgp) [inline app/(landing)/local-elections-webinar/components/LocalElectionsHero.js:67]: missing `priority` here

File: app/(landing)/local-elections-webinar/components/LocalElectionsWebinarPage.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { AcademySignUpModalProvider } from '../../academy/components/AcademySignUpModal/AcademySignUpModalProvider';
+import LocalElectionsHero from './LocalElectionsHero';
+import IsNowYourTime from './IsNowYourTime';
+import CampaignExperts from './CampaignExperts';
+import KickTheTires from './KickTheTires';
+import {
+  DEFAULT_SLANT_SECTION_COLORS,
+  SlantSection,
+} from '@shared/landing-pages/SlantSection';
+import { theme } from '../../../../tailwind.config';
```
Comment (tomer-tgp) [inline app/(landing)/local-elections-webinar/components/LocalElectionsWebinarPage.js:11]: can you import that from its absolute path?

File: app/shared/landing-pages/SlantSection.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { theme } from '../../../tailwind.config';
```
Comment (tomer-tgp) [inline app/shared/landing-pages/SlantSection.js:1]: import from absolute path please

File: app/shared/landing-pages/SlantSection.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+export const DEFAULT_SLANT_SECTION_COLORS = ['rgba(0,0,0,0)', theme.extend.colors.primary.dark, 'rgba(0,0,0,0)'];
+export const SlantSection = ({
```
Comment (tomer-tgp) [inline app/shared/landing-pages/SlantSection.js:4]: nice. There is another variable here - the direction. They change that sometimes

**Review by tomer-tgp — CHANGES_REQUESTED** [review]

File: app/(landing)/local-elections-webinar/components/LocalElectionsHero.js (hunk 1 above)
Comment (RavenHursT) [reply app/(landing)/local-elections-webinar/components/LocalElectionsHero.js:68]: https://github.com/thegoodparty/gp-webapp/pull/158/commits/db925084e9a46a3734d8c7f61eac23c4e040898b

File: app/(landing)/local-elections-webinar/components/LocalElectionsWebinarPage.js (hunk 2 above)
Comment (RavenHursT) [reply app/(landing)/local-elections-webinar/components/LocalElectionsWebinarPage.js]: https://github.com/thegoodparty/gp-webapp/pull/158/commits/0c40f1008de677ce26e1467424ae1f27c4cca7a7

File: app/shared/landing-pages/SlantSection.js (hunk 3 above)
Comment (RavenHursT) [reply app/shared/landing-pages/SlantSection.js]: https://github.com/thegoodparty/gp-webapp/pull/158/commits/0c40f1008de677ce26e1467424ae1f27c4cca7a7

File: app/shared/landing-pages/SlantSection.js (hunk 4 above)
Comment (RavenHursT) [reply app/shared/landing-pages/SlantSection.js:4]: https://github.com/thegoodparty/gp-webapp/pull/158/commits/18fd0b6f205e9587d8d4b37f7847fdbcf29df93a

**Review by tomer-tgp — APPROVED** [review]

**Pull request merged**